boot/bootutil/src/swap_move.c

```diff
@@ -104,8 +104,9 @@ boot_read_image_header(struct boot_loader_state *state, int slot,
 
     /* We only know where the headers are located when bs is valid */
     if (bs != NULL && out_hdr->ih_magic != IMAGE_MAGIC) {
-        rc = -1;
```

Comment:
why -1 is not returned anymore?

Reply:
So you've added the `rc = -1` back to the Move implementation. Shouldn't Swap Scratch also return `-1` in this scenario?